src/App.css

```diff
@@ -2,6 +2,16 @@
   text-align: center;
 }
 
+.appContainer {
+  padding: 10px;
+  width: 80%;
```

Comment:
Does 80% make sense here? That's the width of the app itself. Definitely want some margin but don't know if we need to just not use 20% of the screen.

Reply:
Discussed offline. 80% wrapper should be ok. Resolving.